src/components/Input.tsx

```diff
@@ -105,7 +105,7 @@ class Input extends React.PureComponent<InputProps, InputState> {
 
     const highlightClassName = cx(highlightStyle, {
       _pebble_input_highlight_focused: isFocused,
-      _pebble_input_highlight_state: !!_message,
+      _pebble_input_highlight_state: !!errorMessage || !!successMessage,
```

Comment:
to be frank, I don't like these classnames because they cannot be tree shaken.
No need to change anything now, I am not sure of a solution that would work with emotion 10